Frontend/src/main/java/com/harmonia/client/ServerClient.java

```diff
@@ -7,38 +7,70 @@
 import org.springframework.http.HttpMethod;
 import org.springframework.http.ResponseEntity;
 import org.springframework.http.converter.json.MappingJackson2HttpMessageConverter;
-import org.springframework.web.HttpMediaTypeNotSupportedException;
 import org.springframework.web.client.RestTemplate;
 
-import static com.harmonia.constants.HarmoniaConstants.SERVER_LIST_ALL_URL;
```

Comment:
Removing these imports doesnt affect the usage of the constants in this class?

Reply:
we are already importing all:

`import static com.harmonia.constants.HarmoniaConstants.*;`